rfc#00003

```diff
@@ -0,0 +1,213 @@
+# Title (Ex. 00002: Trust domains and Enarx)
+- Authors: [Mike Bursell](you@github-email)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2020-02-13
+- Status Note: under discussion
+- Start Date: 2020-02-13 (date you started working on this idea)
+- Tags: trust
+- Addresses: (Issue #192)[https://github.com/enarx/enarx/issues/192]
+
+## Summary
+
+Trust domains are a key concept within Enarx.  This RFC provides an
+introduction to trust domains in Enarx.
+
+## Motivation
+
+In order to explain the motivations behind Enarx, our design principles
+and aims, we need to be able to explain what trust domains are.  This RFC
+attempts to provide an introduction.
+
+## Tutorial
+
+See rfc#00002 for an introduction to trust relationships.
+
+### Applying trust relationships to Enarx
+The key trust relationship for Enarx is that held by the tenant
+(workload owner) to that workload.  Enarx aims to provide assurances to
+the tenant that the integrity and confidentiality of that workload are
+protected: these are the key contexts of that trust, and actions can be
+assured if they are maintained.  Trust may decay or be built up in the
+workload - the time context.  In this case, the workload does not have an
+equivalent set of assurances about the tenant, so the relationship is
+not symmetrical.
+
+See rfc#00002 for an introduction to trust relationships.
+
+### Applying trust domains to Enarx - default model
+There are various possible models for trust domains in an Enarx
+deployment - this section describes the simplest, and default model.
+In the default Enarx model, we start off by defining the following parties
+- tenant
+- host owner (the host operator may be separate, but is considered to
+be the same for this description)
+- entities running processes on the host (includes other workloads
+and compromised processes)
+- man-in-the-middle entities
+Clearly, the first two are the key actors - the others are included for
+reference (no threat model is not defined here).
+
+The following components are defined:
+- host 
+- host CPU + CPU firmware
```

Comment:
I think this requires some clarification: what's the difference between the host and the host CPU + CPU firmware.
It may sound obvious to us, but it's actually a not that defined if you think about it:
"host CPU + CPU firmware" is clear: that's the actual CPU that powers the host along with the firmware embedded on this CPU.
"host": is this the rest of the hardware? the software stack (BIOS + bootloader + OS + middleware + all other software running)? both? But wait, it can't include "all other software running" on the host, because there is a "Other workloads" component.

I think while it seems obvious on a instinctive level ("it's the machine which hosts the workloads"), it would benefit from a more precise definition.

Reply:
Very good point.  I've added a description of what the host is.

Reply:
Fixed.